.circleci/config.yml

```diff
@@ -1,76 +1,58 @@
-version: 2
+version: 2.1
 
-#
-# Reusable Snippets!
-#
-# These are re-used by the various tests below, to avoid repetition.
-#
-run_install_desired_npm: &run_install_desired_npm
-  run:
-    name: Install npm@6.
-    command: sudo npm install -g npm@6
+# These "CircleCI Orbs" are reusable bits of configuration that can be shared
+# across projects.  See https://circleci.com/orbs/ for more information.
+orbs:
+  # `oss` is a local reference to the package.  The source for Apollo Orbs can
+  # be found at http://github.com/apollographql/CircleCI-Orbs.
```

Comment:
This comment points to the right place, but I'm just drawing extra attention to it so the source for these can be more easily found.